spec/unit/peep_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'peep'
+
+describe Peep do
+
+  before(:each) do
+    User.create('Ed', 'Ed209', 'password1', 'ed@genericemail.com')
+  end
+
+  describe '.add' do
+
+    it 'adds a peep to the peep feed' do
+      expect { Peep.add('This is a peep') }.to change { Peep.feed.length }.from(0).to(1)
+    end
+
+    it 'will not add a blank peep' do
+      expect { Peep.add('') }.not_to change { Peep.feed.length }
+    end
+  end
+
+  describe '.feed' do
+
+    it 'joins users/peeps databases to create array of peep objects' do
+      Peep.add('This is a peep', '17:00:00')
+      
+      peep = Peep.feed.first
+      
+      expect(peep.handle).to eq '@Ed209'
+      expect(peep.content).to eq 'This is a peep'
+      expect(peep.time).to eq '17:00'
+    end
+
+    it 'sorts array of peeps into reverse chronological order based on time and date' do
```

Comment:
I think there might be too much going on in this test, what you're testing is different to what th `it` is saying.